src/overrides/hooks/tags.py

```diff
@@ -35,24 +35,21 @@ def replace_ai_act(_: Match):
         for type in ["soort-toepassing", "risicogroep", "rol-ai-act", "systeemrisico", "transparantieverplichting"]:
             field = page.meta.get(type, [])
             if isinstance(field, str):
-                if field != "niet-van-toepassing":  # Skip niet-van-toepassing
+                if field != "niet-van-toepassing" and field != "uitzondering-van-toepassing":  # Skip niet-van-toepassing and skip uitzondering van toepassing
                     buttons.append(flag(type, field, page, files))
             else:
                 for role in field:
-                    # Skip niet-van-toepassing
-                    if role == "niet-van-toepassing":
+                    # Skip niet-van-toepassing or uitzondering-van-toepassing
+                    if role == "niet-van-toepassing" or role == "uitzondering-van-toepassing":
                         continue
-                        
-                    # Skip if both 'X' and 'Geen X' exist
-                    if (role.startswith('geen-') and role[5:] in field) or \
-                    (not role.startswith('geen-') and f"geen-{role}" in field):
-                        continue
-                        
+
                     buttons.append(flag(type, role, page, files))
 
-        if len(buttons) == 0:
+        # for non-aia vereisten, remove the buttons and add other toelichting 
+        if not page.meta.get('id').startswith('urn:nl:ak:ver:aia-'):
             toelichting = "Deze vereiste is waarschijnlijk van toepassing op jouw situatie. Controleer de [bronnen](#bronnen) om dit zeker te weten."
```

Comment:
Kunnen we niet stellen dat deze vereisten sowieso gelden? Of is dat te hard? 

```suggestion
            toelichting = "Dit is een vereiste die voor alle algoritmische toepassingen geldt. Controleer de [bronnen](#bronnen) om te kijken of dit ook voor jou van toepassing is."
```

Reply:
Hm dat klopt niet helemaal. Als je bijvoorbeeld geen persoonsgegevens gebruikt, is de AVG waarschijnlijk niet van toepassing

Reply:
Ik heb hem een beetje aangescherpt. Zo beter?